client/Lykke.Service.PayAuth.Client/ApiRunner.cs

```diff
@@ -24,10 +25,39 @@ public async Task<T> RunAsync<T>(Func<Task<T>> method)
             {
                 return await method();
             }
-            catch (ApiException exception)
+            catch (ApiException ex)
             {
-                throw new ErrorResponseException("An error occurred  during calling api", exception);
+                ThrowException(ex);
```

Comment:
There is no sense to return anything after you raise an exception

Reply:
```return default(T);``` is just added to avoid compiler error
![image](https://user-images.githubusercontent.com/11960985/44574862-41c10500-a793-11e8-8f7e-98c07af4d1db.png)

Reply:
stupid compiler )))